prow/plugins/cla/cla.go

```diff
@@ -54,8 +55,13 @@ It may take a couple minutes for the CLA signature to be fully registered; after
 	maxRetries = 5
 )
 
+var (
+	checkCLARe = regexp.MustCompile(`(?mi)^/check-cla\s*$`)
+)
+
 func init() {
 	plugins.RegisterStatusEventHandler(pluginName, handleStatusEvent, helpProvider)
+	plugins.RegisterReviewCommentEventHandler(pluginName, handleCommentEvent, helpProvider)
```

Comment:
Use `RegisterGenericCommentHandler` -- I should be able to activate this plugin with a comment on either a review comment or an issue comment.


https://github.com/kubernetes/test-infra/blob/master/prow/plugins/plugins.go#L114